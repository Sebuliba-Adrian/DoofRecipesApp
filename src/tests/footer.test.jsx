import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from '../components/authentication/footer';

const props = {
  message: 'success',
  link: '/login',
  linkText: 'login',
};

it('renders correctly', () => {
  const footer = renderer
    .create(
    <BrowserRouter>
        <Footer {...props} />
      </BrowserRouter>)
    .toJSON();
  expect(footer).toMatchSnapshot();
});
