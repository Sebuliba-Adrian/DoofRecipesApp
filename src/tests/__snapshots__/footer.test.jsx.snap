// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card mt-2"
>
  <div
    className="card-block"
  >
    <p
      className="card-text text-center message"
    >
      success
      <a
        className="card-link"
        href="/login"
      >
        login
      </a>
    </p>
  </div>
</div>
`;
